doc/qbittorrent.1.md

```diff
@@ -1,36 +1,236 @@
+
+# ENVIRONMENT
+
+_OPTIONS_ may be supplied via environment variables with equivalent names.
+
+For an option named **parameter-name**, the corresponding environment variable name is **QBT\_PARAMETER\_NAME**. In other words, the parameter name is converted to upper-case, any hyphens (**-**) are replaced by underscores (**\_**), and **QBT\_** is prepended to it.
+
+To pass flag values, set the variable to **1** or **TRUE**. For example, **`--add-paused=true`** becomes **QBT\_ADD\_PAUSED=1**
+
+# FILES
+
+## Standard configuration files directory hierarchy
 
-**`--no-splash`** Disables splash screen on startup.
+Currently, qBittorrent complies somewhat with the Freedesktop XDG Base Directory specification. The configuration files are stored in the default directories of the specification:
 
-**`--webui-port=x`** Changes Web UI port to x (default: 8080).
+1. Configuration files are stored in _~/.config/qBittorrent_
+2. Data files are stored in _~/.local/share/data/qBittorrent_ (in the future this will be changed to just _~/.local/share/qBittorrent_, in a backwards-compatible way)
+3. Cache is stored in _~/.cache/qBittorrent_ (currently unused)
 
+However, qBittorrent will not use custom XDG paths set via the **\$XDG\_CONFIG\_HOME** environment variables and the like; it will always use the default ones.
+
+The **`--configuration`** and **`profile`** options will instruct qBittorrent to instead create/use configuration files in different places. Refer to the documentation of each of these options to learn more.
+
+## Files used by qBittorrent
+
+This is a list of the files qBittorrent creates and uses for normal operation. While the name and purpose of these files is unlikely to change, the file formats themselves are not stable, and thus intentionally not documented here. Depending on what features are used, some files may not exist. They are only created when needed.
+
+_~/.config/qBittorrent/qBittorrent.conf_
+
+:   Main configuration file.
+
+_~/.config/qBittorrent/qBittorrent-data.conf_
+
+:   This is where all-time statistics are saved.
+
+_~/.config/qBittorrent/rss/feeds.json_
+
+:    RSS feeds configuration.
+
+_~/.config/qBittorrent/rss/download\_rules.json_
+
+:    RSS auto-downloading rules configuration file.
+
+_~/.local/share/data/qBittorrent/BT\_backup/_
+
+:    This directory contains the _.fastresume_ files, which is the mechanism qBittorrent uses to keep track of the state of each added torrent across restarts. In addition, a copy of each of the added torrents' metainfo files is kept here, with the torrents' hashes as the file names.
+
+_~/.local/share/data/qBittorrent/GeoDB/dbip-country-lite.mmdb_
```

Comment:
maybe omit mentioning the exact file, users knowing the folder existence is enough IMO.

Reply:
@Chocobo1
Exact files are only mentioned if they are very significant on their own, otherwise just the directory is mentioned. For example, files inside the `nova3` directory are not listed. I think all of the files that are explicitly documented right now are relevant/interesting enough to justify that.

But let me know if you have a different opinion.

Reply:
The database name is not guaranteed to be stable (we will change it when the need arise) and as a user, I care less about the actual db name.

Reply:
Fixed.